package.json

```diff
@@ -8,11 +8,12 @@
   ],
   "main": "src/index.js",
   "dependencies": {
-    "bootstrap": "^4.5.0",
-    "react": "^16.12.0",
-    "react-bootstrap": "^1.0.1",
-    "react-dom": "^16.12.0",
-    "react-router-dom": "^5.2.0",
+    "bootstrap": "4.5.0",
```

Comment:
same problem as your last merge. you could leave it now, we will fix it in the last merge.